bgp/gobgp.go

```diff
@@ -457,6 +481,18 @@ func (t *bgpTask) updateRIBs(ctx context.Context) error {
 						//
 						// Note that path.NeighborIp is <nil> for some reason so have to use neigh.
 						t.ribattrs.populateRIBAttrs(path, bgpRIB, neighContainer.GetOrCreateAdjRibOutPre().GetOrCreateRoute(route.Prefix, uint32(i)))
+					}
+				}
+			})
+			t.queryTable(ctx, neigh, "adj-rib-out-post-v6", true, api.TableType_ADJ_OUT, api.Family_AFI_IP6, func(routes []*api.Destination) {
```

Comment:
no action required: would be possible to construct the string from the arguments to this func? it seems like the name is TableType + Family + postPolicy bool

Reply:
Done.